cw_monero/lib/api/wallet.dart

```diff
@@ -66,9 +66,14 @@ String getSeedLegacy(String? language) {
   return legacy;
 }
 
-String getAddress({int accountIndex = 0, int addressIndex = 0}) =>
-    monero.Wallet_address(wptr!,
+String getAddress({int accountIndex = 0, int addressIndex = 0}) {
+  while (monero.Wallet_numSubaddresses(wptr!, accountIndex: accountIndex)-1 < addressIndex) {
+    print("adding subaddress");
+    monero.Wallet_addSubaddress(wptr!, accountIndex: accountIndex);
+  }
```

Comment:
in case when the local state will be different than what's in the monero code, or when we hide excahnge address - then we will just do +1 to the index, and this function should make sure that the address exist (it would return it, but changes to for example label would cause errors)